packages/pn-pa-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -24,6 +24,7 @@ import {
   useIsCancelled,
   useIsMobile,
 } from '@pagopa-pn/pn-commons';
+import { dateIsLessThan10Years } from '@pagopa-pn/pn-commons/src/utility/date.utility';
```

Comment:
@SarahDonvito you should export the new function in the index file of pn-commons, and then using it as anything else imported by pn-commons (eg. see the line above this comment)